### PR TITLE
Feature/delete documents

### PR DESCRIPTION
**Pull request by LeeVoi**
Added the logic to delete documents by selecting a document from the tableview for Admin and ProjectManager

### CODE REVIEW

File: src/main/java/dk/easv/gui/controllers/AdminViewController.java [hunk 1]
```diff
@@ -58,8 +59,14 @@ public void handleLogout() throws IOException {
         this.stage.setTitle("Login");
     }
     @FXML
-    private void handleDelete() {
-
+    private void handleDelete() throws  SQLException{
+        try {
+            Document selectedDocument =  tableView.getSelectionModel().getSelectedValues().get(0);
+            documentModel.deleteDocument(selectedDocument.getId());
+            tableView.getItems().remove(selectedDocument);
```
Comment (patrikvalentiny) [inline src/main/java/dk/easv/gui/controllers/AdminViewController.java]: if you reload the observable list in model you dont need to remove the row from table its gonna do it automatically

File: src/main/java/dk/easv/gui/controllers/ProjectManagerViewController.java [hunk 2]
```diff
@@ -125,6 +125,20 @@ public void logoutAction() throws IOException {
...
     }
+    @FXML
+    public void handleDelete() throws SQLException{
+        if (documentTable == null){ // this condition alert can be removed after adding Documents to the tableview
+            AlertHelper.showDefaultAlert("There is no documents to delete", Alert.AlertType.ERROR);
+        } else {
+            try {
+                Document selectedDocument = documentTable.getSelectionModel().getSelectedValues().get(0);
+                documentModel.deleteDocument(selectedDocument.getId());
+                documentTable.getItems().remove(selectedDocument);
```
Comment (patrikvalentiny) [inline src/main/java/dk/easv/gui/controllers/ProjectManagerViewController.java]: same as above

**Review by patrikvalentiny — DISMISSED** [review]

**Comment by LeeVoi** [comment]
Working on requested changes

**Comment by LeeVoi** [comment]
finished working on requested changes

**Review by patrikvalentiny — APPROVED** [review]

**Pull request merged**